exercises/flatten-array/flatten-array_spec.lua

```diff
@@ -33,4 +33,11 @@ describe('flatten-array', function()
       flatten({ { { {  } } }, { { } } })
     )
   end)
+
+  it('should flatten array from readme example', function()
+    assert.are.same(
+      {1,2,3,4,5},
+      flatten({1,{2,3,nil,4},{null},5})
```

Comment:
understood that the nil/null stuff creates some issues for the implementation (dependent on lua version).

That was the example presented - if the test case is invalid because of these inconsistencies perhaps readme should be modified (or is that inherited from a standard problem template?)